model/src/main/proto/oppia_logger.proto

```diff
@@ -118,6 +277,26 @@ message EventLog {
     // The priority of events whose logs can be removed from the storage if the size exceeds a certain limit
     OPTIONAL = 2;
   }
+
+  // Structure of the generic data.
+  message GenericData {
```

Comment:
This name is a bit too generic. :) Perhaps instead: LearnerDetails might be clearer?

Reply:
Changed this to `LearnerDetailsContext`.